templates/index.html

```diff
@@ -11,7 +11,7 @@
         </nav>
     </header>
     <div>
-      (top-level spec directories will go here)
+      <wpt-results />
```

Comment:
I think that maybe this should be <wpt-results></wpt-results>

(In html5 I think self-closing tags are not a thing; see http://stackoverflow.com/questions/3558119/are-non-void-self-closing-tags-valid-in-html5); there are some "void elements" that don't require closing tags, but custom defined elements are never void elements.